<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputeDialog</class>
 <widget class="QDialog" name="ComputeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>699</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Compute Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="boxDates">
     <property name="title">
      <string>Output Time Period</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0,1">
      <item>
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>Start Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="editStartDate">
        <property name="displayFormat">
         <string>yyyy/MM/dd HH:mm</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_33">
        <property name="text">
         <string>End Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="editEndDate">
        <property name="displayFormat">
         <string>yyyy/MM/dd HH:mm</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="boxModel">
     <property name="title">
      <string>Direct Model</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Permeability:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="editPermeability">
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>m/s</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Porosity:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="editPorosity">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>0.600000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Sediment Thermal Conductivity:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="editSedThCond">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>6.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>kg.m/C.s³</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Sediment Thermal Capacity:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="editSedThCap">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>500.000000000000000</double>
          </property>
          <property name="maximum">
           <double>3000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>m²/C.s²</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Specific Density:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="editDensity">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>1500.000000000000000</double>
          </property>
          <property name="maximum">
           <double>3500.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>kg/m³</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Output Mesh (cm):</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="editOutMesh">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="butOutputMesh">
            <property name="text">
             <string>Edit...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="butModel">
          <property name="text">
           <string>Run Direct Model</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="boxInversion">
     <property name="title">
      <string>Bayesian Inversion</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1,0,1,0,1">
        <item row="1" column="4">
         <widget class="QDoubleSpinBox" name="editPermMax">
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="6">
         <widget class="QDoubleSpinBox" name="editDenStep">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Sediment Thermal Capacity:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Min:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QDoubleSpinBox" name="editCapMin">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>500.000000000000000</double>
          </property>
          <property name="maximum">
           <double>3000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Max:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QDoubleSpinBox" name="editCapMax">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>500.000000000000000</double>
          </property>
          <property name="maximum">
           <double>3000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Max:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Porosity:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Min:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QDoubleSpinBox" name="editPoroMin">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>0.600000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Max:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Max Iterations:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Min:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QDoubleSpinBox" name="editPermStep">
          <property name="decimals">
           <number>7</number>
          </property>
          <property name="maximum">
           <double>0.000100000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="editPermMin">
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Specific Density:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Min:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QDoubleSpinBox" name="editPoroStep">
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="maximum">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Solid Thermal Conductivity:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Min:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="5">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Step:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="6">
         <widget class="QDoubleSpinBox" name="editCapStep">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>500.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QDoubleSpinBox" name="editDenMin">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>1500.000000000000000</double>
          </property>
          <property name="maximum">
           <double>3500.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Max:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QDoubleSpinBox" name="editDenMax">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>1500.000000000000000</double>
          </property>
          <property name="maximum">
           <double>3500.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Step:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Step:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" colspan="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>298</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="4">
         <widget class="QDoubleSpinBox" name="editPoroMax">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>0.600000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Step:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QDoubleSpinBox" name="editCondMin">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>6.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Max:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QDoubleSpinBox" name="editCondMax">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>6.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Step:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QDoubleSpinBox" name="editCondStep">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Permeability:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="spinBox_2">
          <property name="maximum">
           <number>1001</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>488</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="butInversion">
          <property name="text">
           <string>Run Inversion</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::RestoreDefaults</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ComputeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ComputeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxInversion</sender>
   <signal>toggled(bool)</signal>
   <receiver>editDensity</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxInversion</sender>
   <signal>toggled(bool)</signal>
   <receiver>editPermeability</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxInversion</sender>
   <signal>toggled(bool)</signal>
   <receiver>editPorosity</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxInversion</sender>
   <signal>toggled(bool)</signal>
   <receiver>editSedThCap</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxInversion</sender>
   <signal>toggled(bool)</signal>
   <receiver>editSedThCond</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
